### DISCUSSION TITLE
Remove Firstname

### DISCUSSION THREAD

**Discussion by HighError** [zitadel/zitadel#6166] [Q&A]
can i disable using firstname for users?  I would not want to collect this data and force users to enter it

**Comment by hifabienne** [comment]
Hei @HighError 
It is currently not possible to  disable  first and lastname. 
But there is already an issue open to make those fields optional.
As this is a breaking change we need to provide some settings where you can configure what should be required and what not.
I can give you an exact ETA, but it is planned for the current quarter.
https://github.com/zitadel/zitadel/issues/4386
